app/views/additional_details/submissions/form_partials/_phone_form_partial.html.haml

```diff
@@ -1 +1 @@
-= qr.telephone_field :content, class: 'form-control'
+= qr.number_field :content, class: 'form-control'
```

Comment:
Yeah, this was a specific request from Wenjun. She wanted this field to behave exactly like the number field. This may be an issue we should revisit further down the road, though.